antarest/study/business/xpansion_management.py

```diff
@@ -21,23 +21,25 @@
 from antarest.core.exceptions import ChildNotFoundError, LinkNotFound
 from antarest.core.model import JSON
 from antarest.core.serde import AntaresBaseModel
-from antarest.study.business.enum_ignore_case import EnumIgnoreCase
-from antarest.study.business.model.xpansion_model import GetXpansionSettings, XpansionSettingsUpdate
+from antarest.study.business.model.xpansion_model import (
+    GetXpansionSettings,
+    XpansionResourceFileType,
+    XpansionSettingsUpdate,
+)
 from antarest.study.business.study_interface import StudyInterface
 from antarest.study.storage.rawstudy.model.filesystem.factory import FileStudy
 from antarest.study.storage.variantstudy.model.command.create_xpansion_configuration import CreateXpansionConfiguration
 from antarest.study.storage.variantstudy.model.command.remove_xpansion_configuration import RemoveXpansionConfiguration
+from antarest.study.storage.variantstudy.model.command.remove_xpansion_resource import (
+    RemoveXpansionResource,
+    checks_resource_deletion_is_allowed,
+    get_resource_dir,
```

Comment:
a little weird to have it in the "remove" command, maybe we should have a "utils" for xpansion related files ?